packages/utils/regl-renderer/types/rendering/commands/drawMesh/colorOnlyShaders.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const vColorFrag: "\nprecision mediump float;\nuniform vec4 ucolor;\n\nvoid main () {\n  gl_FragColor = ucolor;\n}\n";
```

Comment:
UG… should these be exposed? These shader declarations seem to be internal only.

Reply:
It's exported so it's public. When I auto generated the types it only generates types for exported members.

Reply:
I've removed this since it's not used in another type.